kernel_hardening_checker/engine.py

```diff
@@ -112,6 +112,13 @@ def check(self) -> None:
         else:
             self.result = f'FAIL: "{self.state}"'
 
+        # handle checks, provided with list()
+        if self.expected.startswith('*') and self.expected.endswith('*') and self.state is not None:
```

Comment:
Why do we need `self.state is not None` here?

Reply:
this thing is needed in this case:
```
\n>>>>> checking kconfig number 12 <<<<<
+ coverage run -a --branch bin/kernel-hardening-checker -c ./kernel_hardening_checker/config_files/defconfigs/arm64_defconfig_4.20.config
[+] Kconfig file to check: ./kernel_hardening_checker/config_files/defconfigs/arm64_defconfig_4.20.config
[+] Detected kernel version: (4, 20, 0)
[+] Detected microarchitecture: ARM64
[+] Detected compiler: GCC 80001
Traceback (most recent call last):
  File "/home/vyashnikov/tools/kernel-hardening-checker/bin/kernel-hardening-checker", line 22, in <module>
    kernel_hardening_checker.main()
  File "/home/vyashnikov/tools/kernel-hardening-checker/kernel_hardening_checker/__init__.py", line 475, in main
    perform_checking(mode, kernel_version, args.config, args.cmdline, args.sysctl)
  File "/home/vyashnikov/tools/kernel-hardening-checker/kernel_hardening_checker/__init__.py", line 365, in perform_checking
    perform_checks(config_checklist)
  File "/home/vyashnikov/tools/kernel-hardening-checker/kernel_hardening_checker/engine.py", line 410, in perform_checks
    opt.check()
  File "/home/vyashnikov/tools/kernel-hardening-checker/kernel_hardening_checker/engine.py", line 117, in check
    if self.expected.strip('*') in list(self.state.strip('"').split(',')):
AttributeError: 'NoneType' object has no attribute 'strip'
```